README.md

```diff
@@ -1,9 +1,49 @@
-## Fluent Bit Plugin for Kinesis Streams
+## Fluent Bit Plugin for Amazon Kinesis Data Streams
 
-A Fluent Bit output plugin for Kinesis Streams
+A Fluent Bit output plugin for Amazon Kinesis Data Streams.
 
-# Under Development
+#### Security disclosures
 
-## License
+If you think you’ve found a potential security issue, please do not post it in the Issues.  Instead, please follow the instructions [here](https://aws.amazon.com/security/vulnerability-reporting/) or email AWS security directly at [aws-security@amazon.com](mailto:aws-security@amazon.com).
 
-This library is licensed under the Apache 2.0 License. 
+### Plugin Options
+
+* `region`: The region which your Kinesis stream(s) is/are in.
+* `stream`: The name of the stream that you want log records sent to.
+* `partition_key`: A partition key is used to group data by shard within a stream. A Kinesis Data Stream uses the partition key that is associated with each data record to determine which shard a given data record belongs to. For example, if your logs come from Docker containers, you can use container_id as the partition key, and the logs will be grouped and stored on different shards depending upon the id of the container they were generated from. As the data within a shard are coarsely ordered, you will get all your logs from one container in one shard roughly in order. If you don't set a partition key or put an invalid one, a random key will be generated, and the logs will be directed to random shards. In case of invalid partition key, the plugin will print an warning message.
+* `data_keys`: By default, the whole log record will be sent to Kinesis. If you specify a key name(s) with this option, then only those keys and values will be sent to Kinesis. For example, if you are using the Fluentd Docker log driver, you can specify `data_keys log` and only the log message will be sent to Kinesis. If you specify multiple keys, they should be comma delimited.
+* `role_arn`: ARN of an IAM role to assume (for cross account access).
+* `endpoint`: Specify a custom endpoint for the Kinesis Streams API.
+
+### Permissions
+
+The plugin requires `kinesis:PutRecords` permissions.
+
+### Credentials
+
+This plugin uses the AWS SDK Go, and uses its [default credential provider chain](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html). If you are using the plugin on Amazon EC2 or Amazon ECS, the plugin will use your EC2 instance role or ECS Task role permissions. The plugin can also retrieve credentials from a (shared credentials file)[https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html], or from the standard `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` environment variables.
```

Comment:
if we are using the default credential provider chain, maybe we should just link to that and don't explain it again here.